tests/integration/graphql_test.ts

```diff
@@ -1,5 +1,4 @@
 import { Rhum } from "../deps.ts";
```

Comment:
Wait wasn’t this used to create the quick schema in the test?

Reply:
this is now `GraphQL.buildSchema`. instead of just importing parts of the GraphQL library, i'm importing all of it and exporting it from `deps.ts`. this way, users can use all of GraphQL instead of only the things we import/export